empyrical/stats.py

```diff
@@ -714,7 +714,8 @@ def beta(returns, factor_returns, risk_free=0.0):
     if np.absolute(joint.var().iloc[1]) < 1.0e-30:
         return np.nan
 
-    return np.cov(joint.values.T, ddof=0)[0, 1] / np.var(joint.iloc[:, 1])
+    cov = np.cov(joint.values.T, ddof=0)
```

Comment:
Yes, and there's a pr to reuse the covariance matrix values
https://github.com/quantopian/empyrical/pull/30/commits/19cc30e16b32bae4797ff56079e967a2e14ba172